internal/tools/terraform/plan.go

```diff
@@ -27,11 +30,16 @@ func (c *Cmd) Init(remoteKey string) error {
 		return errors.Wrap(err, "unable to write terraform backend state file")
 	}
 
+	awsRegion := os.Getenv("AWS_REGION")
+	if awsRegion == "" {
+		awsRegion = toolsAWS.DefaultAWSRegion
+	}
+
 	_, _, err = c.run(
 		"init",
 		arg("backend-config", fmt.Sprintf("bucket=%s", c.remoteStateBucket)),
 		arg("backend-config", fmt.Sprintf("key=%s/%s", remoteStateDirectory, remoteKey)),
-		arg("backend-config", "region=us-east-1"),
```

Comment:
@cpanato maybe to leave that one at `us-east-1` as the `-backend-config=bucket=mattermost-kops-state-test` exists in us-east-1 or we can have a different approach on using different bucket on other regions

Reply:
i think we need to have a different bucket for other regions as well to not mess up